RELEASE-NOTES.txt

```diff
@@ -4,6 +4,10 @@ Unreleased
 * [*] Tablet view fixes for inserter button. https://github.com/wordpress-mobile/gutenberg-mobile/pull/3602
 * [**] Fixed an issue where pressing enter inside a text-based block was not creating a new block when using Gboard [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3590]
 
+1.55.2
+------
+* [**] Fix incorrect block insertion point after blurring the post title field. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3640]
```

Comment:
I think we have this entry duplicated in the `Unreleased` section so we could remove it from there.

Reply:
Good catch. Addressed in https://github.com/wordpress-mobile/gutenberg-mobile/pull/3655/commits/5ba6364e562714cf9d23e242be69ce243dc0bc2b.